src/lib/components/DatePickerPanel.js

```diff
@@ -37,11 +38,7 @@ export default class DatePickerPanel extends React.Component {
     const { currentValue, confirmDisabled } = this.state;
     const disabledTimeFns = this.disabledTime();
 
-    const uiLocale = [browser.i18n.getUILanguage().replace('_', '-'), 'en-US'];
-    const dtf = new Intl.DateTimeFormat(uiLocale, {hour: 'numeric'});
-    const timeFormat = (dtf.resolvedOptions().hour12)
-      ? 'h:mm a'
-      : 'HH:MM';
+    const timeFormat = use12hFormat ? 'h:mm a' : 'HH:mm';
```

Comment:
I think we could move the `?:` part into time-formats, too, and just export a `timeFormat` constant from that file into this file…